Update virtual-networks-name-resolution-for-vms-and-role-instances.md

cristhianu opened the pull request:
A recent change started preventing users of Azure App Service using VNET Integration to be able to resolve Azure VM's internal names. Since this wasn't supposed to be allowed in the first place, we need to have it documented so that we can explain to customer what the workaround is.

azurecla commented:
Hi __@jaymathe__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaymathe). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

cristhianu committed:
Update virtual-networks-name-resolution-for-vms-and-role-instances.md

A recent change started preventing users of Azure App Service using VNET Integration to be able to resolve Azure VM's internal names. Since this wasn't supposed to be allowed since the beginning we need to have it document it so that we can explain to customer what the workaround is.
